java-spring/rest/jv-spring-rest_checklist.md

```diff
@@ -1,13 +1,14 @@
-# Spring REST
+## Common mistakes (jv-spring-rest)
 
-* In order to properly use new dependencies in your project, you might need to run `mvn clean package`.
 * Make your code consistent. Use the @Autowired annotation everywhere with the constructors or delete it from everywhere.
 * Return `id` of entity in all response DTOs.
 * Remember about [REST resource-naming convention](https://restfulapi.net/resource-naming/).
 * In your mappers prefer setters over big constructors while converting entity to dto and vice versa.
 * Use Stream API while working with lists in your mappers.
-* In your MovieSessionRequest/ResponseDtos, we don't need to pass the whole information about Movie and CinemaHall - 
+* In your MovieSessionRequest/ResponseDtos, you don't need to pass the whole information about Movie and CinemaHall - 
```

Comment:
```suggestion
* In your MovieSessionRequest/ResponseDtos, we don't need to pass the whole information about Movie and CinemaHall - 
```
this way we use more formal style

Reply:
It seems to me, 'we' does not look consistent in this place. Are you sure with your comment?
![image](https://user-images.githubusercontent.com/20326858/226344781-5b32bb5e-6e22-468e-a71e-cbadac59ab79.png)

Reply:
agree, not consistent
then here it is up to you to change you->we, your->our everywhere to be more formal
or change we->you